### PR TITLE
Fix `uniqueTokenManagementSpecs` leftover tests

### PR DESCRIPTION
**Pull request by georgi-l95**
**Description**:
This PR fixes tests from the `uniqueTokenManagementSpecs`

**Related issue(s)**:

Fixes #

**Notes for reviewer**:
One of the tests was change to expect `OK` status on precheck instead, because `OK` is only being used on precheck step.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenAccountWipeHandlerTest.java [hunk 1]
```diff
@@ -631,7 +632,7 @@ void nftSerialNumsIsEmpty() {
 
             assertThatThrownBy(() -> subject.handle(context))
                     .isInstanceOf(HandleException.class)
-                    .has(responseCode(INVALID_WIPING_AMOUNT));
+                    .has(responseCode(FAIL_INVALID));
```
Comment (iwsimon) [inline hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenAccountWipeHandlerTest.java:635]: Why did you change the test response code here?

Comment (georgi-l95) [reply hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenAccountWipeHandlerTest.java:635]: It's expected in the HAPI test to fail for this with FAIL_INVALID. And with the changes in the handler this is how it fails. Can't say for sure what's the correct error, but I treat the HAPI Tests as source of truth 😄

Comment (iwsimon) [reply hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/TokenAccountWipeHandlerTest.java:635]: This is just unit test. Np.

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/UniqueTokenManagementSpecs.java [hunk 2]
```diff
@@ -767,7 +771,7 @@ private HapiSpec uniqueWipeFailsWhenInvokedOnFungibleToken() { // invokes unique
                                 .hasKnownStatus(INVALID_WIPING_AMOUNT)
                                 .via("wipeTx"),
                         wipeTokenAccount(A_TOKEN, ACCOUNT, List.of())
-                                .hasKnownStatus(OK)
+                                .hasPrecheck(OK)
```
[the remark this answers is not in the document]
Comment (georgi-l95) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/UniqueTokenManagementSpecs.java:774]: It's passing on this step, but failing on another, which has nothing to do with my changes.

[the remark this answers is not in the document]
Comment (iwsimon) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/UniqueTokenManagementSpecs.java:774]: Were you saying this test method failed on the mono code run? Then we don't have to enable HapiTest for this method.

[the remark this answers is not in the document]
Comment (georgi-l95) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/UniqueTokenManagementSpecs.java]: Okay, i've disabled it and revert the change, we can revisit it at later point.

**Review by tinker-michaelj — APPROVED** [review]
Unfortunate that we have a use of `FAIL_INVALID` that doesn't imply an internal error, but no reason to change it _just now_. 😁 

LGTM, thanks @georgi-l95 !
